README.md

```diff
@@ -1,2 +1,91 @@
 # terraform-provider
 Ably Terraform Provider
+
+# [Ably](https://www.ably.com)
+
+_[Ably](https://ably.com) is the platform that powers synchronized digital experiences in realtime. Whether attending an event in a virtual venue, receiving realtime financial information, or monitoring live car performance data – consumers simply expect realtime digital experiences as standard. Ably provides a suite of APIs to build, extend, and deliver powerful digital experiences in realtime for more than 250 million devices across 80 countries each month. Organizations like Bloomberg, HubSpot, Verizon, and Hopin depend on Ably’s platform to offload the growing complexity of business-critical realtime data synchronization at global scale. For more information, see the [Ably documentation](https://ably.com/docs)._
+
+[![npm version](https://img.shields.io/npm/v/ably.svg?style=flat)](https://img.shields.io/npm/v/ably.svg?style=flat)
+
+This is a Terraform provider for Ably.
+
+## Supported platforms
+
+This provider supports the following systems/architectures:
+
+- Darwin / AMD64
+- Darwin / ARMv8
+- Linux / AMD64 (this is required for usage in Terraform Cloud, see below)
+- Linux / ARMv8 (sometimes referred to as AArch64 or ARM64)
+- Linux / ARMv6
+- Windows / AMD64
+
+## Installation
+
+To install Ably Terraform provider, add the following to your Terraform configuration file: 
+
+```terraform
+terraform {
+  required_providers {
+    ably = {
+      source = "ably/ably"
+      version = "0.0.0"
```

Comment:
Shouldn't we document that a token can also be passed into the config?

Reply:
This is a good point. I think we should give example of the recommended approach (with passing the token via environment variable), and then in documentation user will be able to learn about alternative method. As this alternative method is less secure I think it will be better if we will not show it straightaway.

Reply:
And we already mention that it is possible to pass token directly in the comment in the code snippet anyway.

Reply:
After thinking updated the readme to make the initialisation process clearer.

Reply:
After additional chat removed mentions of custom endpoints.